tapglue-android-sdk/src/main/java/com/tapglue/model/TGSocialId.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2015 Tapglue (https://www.tapglue.com/). All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.tapglue.model;
+
+/**
+ * This is a helper class, therefore don't extend TGBaseObject
+ */
+public class TGSocialId {
```

Comment:
Please remove this and rework accordingly.
The social ID is a string, a bug which I've reported in a separate issue: https://github.com/tapglue/android_sdk/issues/9 
Please fix that one first, then change the code base to use: String socialPlatform, Array<String> socialIDs

Thank you.